packages/http-client-js/package.json

```diff
@@ -32,6 +32,10 @@
     "./testing": {
       "types": "./dist/src/testing/index.d.ts",
       "default": "./dist/src/testing/index.js"
+    },
+    "./experimental": {
```

Comment:
Since this package is preview I don't think we need an experimental sub export. However I think we can export these under components.

import { Client } from "@typespec/http-client-js/components"